modules/ROOT/pages/api-gateway-encryption-mule4.adoc

```diff
@@ -0,0 +1,270 @@
+
+You can configure your gateway logger to display information about the encryption mechanism during startup.
+
+In `$MULE_HOME/conf/log4j2.xml`, set up an asynchronous logger for your gateway deployment:
+
+[source,XML,linenums]
+--
+...
+  <AsyncLogger name="com.mulesoft.mule.runtime.gw.deployment" level="DEBUG" />
+...
+--
+
+When you provide a key, the runtime displays the following message:
+
+----
+An encryption key is present. Policies and API contracts will be encrypted.
+----
+
+If not, the runtime stars with:
+
+----
+No encryption key provided. Policies and API contracts won't be encrypted.
+----
+
+If the decryption process fails, the runtime displays an error:
+
+----
+Client ID cannot be decrypted
+----
+
+If your credentials can't be decrypted, you cannot establish a connection with Platform, and won't be able to download policies or contracts.
+
+== Troubleshooting Credentials
+
+If you run into issues when passing your encrypted credentials to Anypoint Platform, it's recommended to check that the provided credentials are valid. You can use the Secure Properties Tool described earlier to decrypt the values back to the originals and compare them with the ones displayed in the platform:
+
+. Run the Secure Properties Tool to decrypt your credentials:
++
+[source]
+--
+> java -jar secure-properties-tool.jar string decrypt AES CBC <yourEncryptionKey> <>
+--
++
+. Compare the decrypted value with the one in Anypoint Platform. +
+See xref:org-credentials-config-mule4.adoc#obtaining-credentials[Obtaining Credentials] for information on how to verify your platform's credentials.
+
+== Supported Properties
+
+The following `wrapper.conf` properties support encryption:
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+Property | | Description
+| Client ID | `anypoint.platform.client_id.<n>` | Your Platform Client ID
+| Client Secret | `anypoint.platform.client_secet.<n>` | Your Platform Client Secret
+| Proxy Password | `anypoint.platform.proxy_password.<n>` | Password of the proxy configuration to connect to Platform
+|===
+
+== Policies Encryption
+
+The configuration for your API policies is stored in the `policies` directory, and some policies may contain sensitive information that you want to hide from unauthorized parties, such as the password field for the `Basic Authentication` header. +
+Sensitive values in your provided API policies are marked as such in the API Manager UI for policy configuration. When creating a custom policy, you can define which values are sensitive.
+
+== Configuring Policy Encryption
+
+To enable encryption in policies, you need to provide two system properties in your `wrapper.conf` file:
+
+* `anypoint.platform.encryption_key.<n>`. +
+The encryption key for your policy.
++
+[CAUTION]
+--
+If you already have all your policies encrypted and then needed to restart the runtime without an encryption key, all policies will be downloaded again, and all the sensitive information won't be encrypted.
```

Comment:
This sentence sounds misleading, it is trying to say the following: 
"If you have all your policies encrypted, and then you restart Mule without an encryption key,..." ?